src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        Lottery ball2 = new Lottery();
+        Lottery ball3 = new Lottery();
+        System.out.println(ball1.getRandomBall());
+        System.out.println(ball2.getRandomBall());
+        System.out.println(ball3.getRandomBall());
```

Comment:
1. Please use cycle for repetitive actions
2. Single lottery object is more than enough, you should be able to call .getRandomBall() method from one Lottery object as many times as required